v0/destinations/ga4/data/GA4SpendVirtualCurrencyConfig.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Why it is empty? Is it added due to remaining consistent, like for other events we are using a config.json we should use one for this event also?

Reply:
Added. It got left out.